tests/integration/icontracts/conftest.py

```
@@ -61,12 +120,12 @@ def _setup(mock_response=None):
-def mock_llms():
+def mock_llms() -> bool:
+    """Return True when mock LLM mode is enabled via TEST_WITH_MOCK_LLMS=true."""
     env_var = os.getenv("TEST_WITH_MOCK_LLMS", "false")  # default no mocking
-    if env_var == "true":
-        return True
-    return False
+    return env_var == "true"
 
 
-def pytest_configure(config):
+def pytest_configure(config: Any) -> None:
```

Ruff (0.15.2)

[warning] Unused function argument: `config`

(ARG001)